lib/network/api/resource/const.go

```diff
@@ -0,0 +1,7 @@
+package resource
```

Comment:
We already use the similar file name, `/lib/constant.go`. Using same name will be good, I think.